docs/diagrams/ArchitectureSequenceDisgram.puml

```diff
@@ -0,0 +1,38 @@
+@startuml
```

Comment:
@omupenguin , yetong1895 meant the naming of this puml file

Reply:
oh, thanks for the catch!